.pax/pre-packaging.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh -e
+set -x
+
+################################################################################
+# This program and the accompanying materials are made available under the terms of the
+# Eclipse Public License v2.0 which accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright IBM Corporation 2018, 2020
+################################################################################
+
+FUNC=[CreatePax][pre-packaging]
+PWD=$(pwd)
+
+# npm install on z/OS to get the keyring_js node modules
+cd ./content/server
+npm install
```

Comment:
We can avoid adding dev dependencies into the package.

Reply:
The build fails without the dev dependencies as they have to be done on z/OS???